src/Groups/homomorphisms.jl

```diff
@@ -28,9 +28,9 @@ export
 
 function Base.show(io::IO, x::GAPGroupHomomorphism)
   print(io, "Group homomorphism from \n")
-  println(io, domain(x))
-  print(io, "to\n")
-  println(io, codomain(x))
+  show(IOContext(io, :compact => true), domain(x))
+  print(io, "\nto\n")
+  show(IOContext(io, :compact => true), codomain(x))
```

Comment:
Currently there seem to be no `show` methods for the groups in question that honor the `:compact` flag. Is this something for later?

Reply:
Matrix groups use the `:compact` flag. This is because the name of a matrix group could be quite long, so to print a homomorphism it seems better to print just the name of the variable identifying the domain/codomain whenever this is a matrix group. But, of course, we can change idea and leave the name of the group. Or, do the opposite and extend it to the other types of groups.